### PR TITLE
added documentation about cluster configuration

### PR DESCRIPTION
**Pull request by anistal**
#### Description
added documentation about cluster configuration

### CODE REVIEW

File: doc/src/site/sphinx/cluster-config.rst [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+| executionJarFolder | Hdfs path that contains a jar with the driver that will run in    |
+|                    | the cluster.                                                      |
++--------------------+-------------------------------------------------------------------+
+| classpathFolder    | Hdfs path that contains other needed jars by the driver.          |
++--------------------+-------------------------------------------------------------------+
+
+Sparkta + Spark Standalone
+==========================
+
+You must have correctly configured and deployed Spark StandAlone. You can obtain information about how to do it
```
Comment (danielcsant) [inline doc/src/site/sphinx/cluster-config.rst]: Standalone

File: doc/src/site/sphinx/cluster-config.rst [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+   target="_blank">here</a>
+
+The next step is set up Sparkta's Yarn configuration:
+
+  yarn {
+  }
+
++--------------------------------+-------------------------------------------------------------------+
+| Property                       | Description                                                       |
++================================+===================================================================+
```
Comment (danielcsant) [inline doc/src/site/sphinx/cluster-config.rst]: Is this missed?

Comment (danielcsant) [inline doc/src/site/sphinx/cluster-config.rst]: General configuration should be here as well

Comment (anistal) [inline doc/src/site/sphinx/cluster-config.rst]: 1. I have not seen configuration about Yarn.
2. What do you mean about general configuration? Maybe this ...

 config {
    executionMode = local
    rememberPartitioner = true
    topGracefully = false
  }

File: doc/src/site/sphinx/configuration.rst [hunk 3]
```diff
@@ -0,0 +1,307 @@
+Sparkta Configuration
+*********************
+
+Overview
+========
+
+This section shows how to configure Sparkta through Typesafe's Config standard.
+
+By default the system will use the config file "reference.conf", but you can overwrite this values having and
+application.conf in your classpath. This is resume of all parameters that you can configure:
```
Comment (danielcsant) [inline doc/src/site/sphinx/configuration.rst]: **resume** is a false friend ;) It should be `This is the summary of all parameters that you can configure`

File: doc/src/site/sphinx/configuration.rst [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+   * :ref:`cluster-config-yarn`
+
+More deep documentation about it, can be found in |typesafe_config_repo|.
+
+.. |typesafe_config_repo| raw:: html
+
+   <a href="https://github.com/typesafehub/config"
+   target="_blank">Github Typesafe's repository</a>
+
+A reference.conf with a full example of a configuration can be found in |github_sparkta_repo_reference_conf|.
```
Comment (danielcsant) [inline doc/src/site/sphinx/configuration.rst]: `can be found at`

File: doc/src/site/sphinx/configuration.rst [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+
+.. _spray-config:
+
+Spray.io configuration
+======================
+
+Spray is a toolkit for building REST/HTTP-based integration layers on top of Scala and Akka and is the main door to
+offer a Sparkta API to the world.
+
+More deep documentation about it, can be found in |spray_io|.
```
Comment (danielcsant) [inline doc/src/site/sphinx/configuration.rst]: `can be found at`

File: doc/src/site/sphinx/configuration.rst [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+  spray.can {
+    verbose-error-messages = on
+  }
+
+.. _api-config:
+
+API configuration
+=================
+
+Parameters related with the API REST.
```
Comment (danielcsant) [inline doc/src/site/sphinx/configuration.rst]: `related to`

File: doc/src/site/sphinx/configuration.rst [hunk 7]
```diff
@@ -0,0 +1,307 @@
...
++----------+----------------------------------------+
+
+.. _swagger-config:
+
+Swagger configuration
+=====================
+
+Swagger is a Spray.io's submodule that it is used to generate automatically documentation about API's endpoints.
+
+More deep documentation about it, can be found in |swagger|.
```
Comment (danielcsant) [inline doc/src/site/sphinx/configuration.rst]: `can be found at`

File: doc/src/site/sphinx/configuration.rst [hunk 8]
```diff
@@ -0,0 +1,307 @@
...
+
+From the client, you can submit a job to Mesos cluster by running spark-submit and specifying the master url to the url
+of the MesosClusterDispatcher (e.g: mesos://dispatcher:7077). You can view driver statuses on the Spark cluster Web UI.
+
+.. _cluster-config-yarn:
+
+Sparkta + Apache YARN
+---------------------
+
+You must have correctly configured and deployed Apache Yarn. You can obtain information about how to do it in
```
Comment (danielcsant) [inline doc/src/site/sphinx/configuration.rst]: `YARN`

Comment (danielcsant) [inline doc/src/site/sphinx/configuration.rst]: `how to do it at`

File: doc/src/site/sphinx/configuration.rst [hunk 9]
```diff
@@ -0,0 +1,307 @@
...
+|yarn_doc|.
+
+.. |yarn_doc| raw:: html
+
+   <a href="https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/YARN.html"
+   target="_blank">here</a>
+
+The next step is set up Sparkta's YARN configuration:
```
Comment (danielcsant) [inline doc/src/site/sphinx/configuration.rst]: Please remove from this line to the bottom. This doc goes to master branch so it shouldn't be uncompleted

**Pull request merged**